library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -65,6 +71,8 @@
     private CalendarDay currentMonth;
     private TitleFormatter titleFormatter = DEFAULT_TITLE_FORMATTER;
 
+    Set<DayViewDecorator> dayViewDecorators;
```

Comment:
We should use a List instead of a Set. Then we can define the order in which decorators are applied (and a hash for DayViewDecorator isn't going to be predictable enough for a HashSet). If you just switch to List, we'd have first added first applied, which I think is best.